CMakeLists.txt

```diff
@@ -678,6 +688,8 @@ set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_FULL_LIBDIR}/pkgconfig"
 
 add_library(${PROJECT_NAME} ${MINIZIP_SRC} ${MINIZIP_HDR})
 
+add_library(MINIZIP::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
```

Comment:
Can you wrap this in: `if(NOT TARGET MINIZIP::${PROJECT_NAME})` in case user defines their own.

Then also call:
```
 set_target_properties(MINIZIP::${PROJECT_NAME} PROPERTIES
        INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR}");
```
That way if anybody links against `MINIZIP::${PROJECT_NAME}` they don't have to specify include dirs because it is already done. What do you think?